File: C++/3232. Find if Digit Game Can Be Won.cpp
class Solution {
public:
    bool canAliceWin(vector<int>& n) {
        int i,s=0,r=0;
        for(i=0;i<n.size();i++)
        {
            if(n[i]<10)
            s+=n[i];
            else
            r+=n[i];
        }
        if(r==s)
        return false;
        else
        return true;
    }
};

File: JAVA/3232. Find if Digit Game Can Be Won.java
class Solution {
    public boolean canAliceWin(int[] n) {
        int s=0,r=0;
        for(int i=0;i<n.length;i++){
            if(n[i]<10)
            s+=n[i];
            else
            r+=n[i];
        }
        return r!=s;
    }
}